src/main/java/au/id/micolous/metrodroid/card/CardReader.kt

```diff
@@ -64,7 +65,8 @@ object CardReader {
             val transceiver = AndroidFelicaTransceiver(tag)
 
             transceiver.connect()
-            val c = FelicaReader.dumpTag(transceiver, feedbackInterface)
+            val c = FelicaReader.dumpTag(
+                transceiver, feedbackInterface, onlyFirst = Preferences.felicaOnlyFirst)
```

Comment:
Why do this on Android? I can already see the bug reports from users who misconfigured this

Reply:
That's for debug purposes, but I thought the string makes the impact pretty clear.  I put this in "developer options" rather than "NFC settings" to reflect this.  But one can also mess up MIFARE Classic reading by adjusting the retry value -- either making it extremely slow to fail, or more likely to have read failures.